src/stories/components/BaseAlert.stories.mdx

```diff
@@ -93,4 +103,77 @@ import BaseAlert from '../../components/BaseAlert.vue';
         </div>`
     }}
   </Story>
+</Preview>
+
+# Inline Variants
+
+<Preview>
+  <Story name="Inline Variants">
+    {{
+      components: {
+        BaseAlert,
+        ChecIcon,
+      },
+      data() {
+        return {
+          variants: ['success', 'error', 'warning', 'info'],
+          closed: [],
+        };
+      },
+      methods: {
+        close(variant) {
+          if (!this.isClosed(variant)) {
+            this.closed.push(variant);
+            setTimeout(() => {
+              this.closed.splice(this.closed.indexOf(variant), 1);
+            }, 3000)
+          }
+        },
+        isClosed(variant) {
+          return this.closed.includes(variant);
+        },
+        content(variant) {
+          return `${variant.substring(0, 1).toUpperCase()}${variant.substring(1)} variant`;
+        },
+        icon(variant) {
+          if (variant == 'success') {
+            return 'check-square';
+          }
+          if (variant == 'error') {
+            return 'close-square';
+          }
+          if (variant == 'warning') {
+            return 'exclamation-square';
+          }
+          if (variant == 'info') {
+            return 'info-square';
+          }
+          return 'info-square';
```

Comment:
What do you think about making this stuff part of the component - as a `show-icon` prop?